### DISCUSSION TITLE
Cron schedule on multiple instances for HA architecture like active/passive.

### DISCUSSION THREAD

**Discussion by sarashino** [directus/directus#17789] [Archive]
Currently, a cron scheduled job runs multiple times when multiple instance is standby as active/passive HA clustering.
To ensure that it runs once, lock flag is necessary imho.

Are there any workaround...?

**Comment by sarashino** [comment]
perhaps, we could check mutex via update operation with where clauses

**Comment by BrianJM** [comment]
#15052 is the primary issue for cron schedules in HA environments.
